demo/devserver.js

```diff
@@ -0,0 +1 @@
+window.PushStateTree = require('push-state-tree');
```

Comment:
I think it's a good point, need to add some comment here to explain why I'm doing this.